### DISCUSSION TITLE
测延迟很慢

### DISCUSSION THREAD

**Discussion by zcm931014** [2dust/v2rayNG#3856] [Q&A]
不知各位有没有觉得，v2rayng测节点延迟很慢，要是节点稍微多一点，要很久才能把全部节点的真连接延迟测完，大佬能不能优化下。

**Comment by 2dust** [comment]
考虑到手机系统限制，测试时是开10个线程并发，因为太多了可能会被系统杀掉

**Reply by zcm931014** [reply]
大佬，能不能增加一个设置选项自定义延迟并发线程。

**Reply by p9876000j** [reply]
> 考虑到手机系统限制，测试时是开10个线程并发，因为太多了可能会被系统杀掉

hello，测试真连接在1.8版本后更新过吗，我有个手机好像无法测试
